e2e/testcafe-devextreme/tests/dataGrid/keyboardNavigation/editOnKeyPress.screenshots.ts

```diff
@@ -62,3 +64,75 @@ const DATA_GRID_SELECTOR = '#container';
     await makeRowsViewTemplatesAsync(DATA_GRID_SELECTOR);
   });
 });
+
+test('Focused cell should not flick (T1206435)', async (t) => {
+  type TestWindow = (typeof window) & {
+    counter?: number;
+  };
+
+  const dataGrid = new DataGrid(DATA_GRID_SELECTOR);
+  const firstCell = dataGrid.getDataCell(0, 0).element;
+  const secondCell = dataGrid.getDataCell(1, 0).element;
+  const getFocusEventCount = ClientFunction(
+    () => (window as TestWindow).counter,
+  );
+
+  await ClientFunction(() => {
+    const testWindow = window as TestWindow;
+    testWindow.counter = 0;
+    (secondCell() as any as HTMLElement).addEventListener('focusin', () => {
+      testWindow.counter! += 1;
+    });
+  }, {
+    dependencies: { secondCell },
+  })();
+
+  await t.click(firstCell);
+
+  await t.pressKey('M');
+  await t.pressKey('enter');
+
+  await t.expect(secondCell.focused).ok();
+
+  const focusEventCount = await getFocusEventCount();
+  await t.expect(focusEventCount).eql(1);
+
+  await ClientFunction(() => {
+    delete (window as TestWindow).counter;
+  })();
+}).before(async () => {
+  await createWidget('dxDataGrid', ClientFunction(() => {
+    const data = [
+      { value: 'data' },
+      { value: 'data' },
+    ];
+    return {
+      dataSource: new (window as any).DevExpress.data.CustomStore({
+        load() {
+          return Promise.resolve(data);
+        },
+        update() {
+          return new Promise<void>((res) => {
+            setTimeout(() => {
+              res();
+            }, 100);
+          });
+        },
+      }),
+      keyboardNavigation: {
+        enabled: true,
+        editOnKeyPress: true,
+        enterKeyAction: 'moveFocus',
+        enterKeyDirection: 'column',
+      },
+      editing: {
+        mode: 'cell',
+        allowUpdating: true,
+        allowAdding: true,
+        startEditAction: 'dblClick',
+        refreshMode: 'reshape',
+      },
+      repaintChangesOnly: true,
+    } satisfies Properties;
```

Comment:
Let's remove the ```satisfies Properties``` or change it to ```as Properties```
This test cannot be run locally, the following TS error appears (I checked deps and reinstall all of them):

```
ERROR Cannot prepare tests due to the following error:

Error: TypeScript compilation failed.
/Users/william/work/checks/devextreme_Roman/e2e/testcafe-devextreme/tests/dataGrid/keyboardNavigation/editOnKeyPress.screenshots.ts (104, 36): Argument of type 'ClientFunction<{ dataSource: any; keyboardNavigation: { enabled: boolean; editOnKeyPress: boolean; enterKeyAction: any; enterKeyDirection: string; }; editing: { mode: string; allowUpdating: boolean; allowAdding: boolean; startEditAction: string; refreshMode: string; }; repaintChangesOnly: boolean; }, []>' is not assignable to parameter of type 'Properties<any, any> | (() => Properties<any, any> | Promise<Properties<any, any>>)'.
  Type 'ClientFunction<{ dataSource: any; keyboardNavigation: { enabled: boolean; editOnKeyPress: boolean; enterKeyAction: any; enterKeyDirection: string; }; editing: { mode: string; allowUpdating: boolean; allowAdding: boolean; startEditAction: string; refreshMode: string; }; repaintChangesOnly: boolean; }, []>' is not assignable to type '() => Properties<any, any> | Promise<Properties<any, any>>'.
    Type 'Promise<{ dataSource: any; keyboardNavigation: { enabled: boolean; editOnKeyPress: boolean; enterKeyAction: any; enterKeyDirection: string; }; editing: { mode: string; allowUpdating: boolean; allowAdding: boolean; startEditAction: string; refreshMode: string; }; repaintChangesOnly: boolean; }>' is not assignable to type 'Properties<any, any> | Promise<Properties<any, any>>'.
      Type 'Promise<{ dataSource: any; keyboardNavigation: { enabled: boolean; editOnKeyPress: boolean; enterKeyAction: any; enterKeyDirection: string; }; editing: { mode: string; allowUpdating: boolean; allowAdding: boolean; startEditAction: string; refreshMode: string; }; repaintChangesOnly: boolean; }>' is not assignable to type 'Promise<Properties<any, any>>'.
        Type '{ dataSource: any; keyboardNavigation: { enabled: boolean; editOnKeyPress: boolean; enterKeyAction: any; enterKeyDirection: string; }; editing: { mode: string; allowUpdating: boolean; allowAdding: boolean; startEditAction: string; refreshMode: string; }; repaintChangesOnly: boolean; }' is not assignable to type 'Properties<any, any>'.
          Type '{ dataSource: any; keyboardNavigation: { enabled: boolean; editOnKeyPress: boolean; enterKeyAction: any; enterKeyDirection: string; }; editing: { mode: string; allowUpdating: boolean; allowAdding: boolean; startEditAction: string; refreshMode: string; }; repaintChangesOnly: boolean; }' is not assignable to type 'Omit<GridBaseOptions<dxDataGrid<any, any>, any, any>, OverriddenKeys>'.
            The types of 'keyboardNavigation.enterKeyDirection' are incompatible between these types.
              Type 'string' is not assignable to type 'EnterKeyDirection'.
/Users/william/work/checks/devextreme_Roman/e2e/testcafe-devextreme/tests/dataGrid/keyboardNavigation/editOnKeyPress.screenshots.ts (136, 7): ';' expected.
/Users/william/work/checks/devextreme_Roman/e2e/testcafe-devextreme/tests/dataGrid/keyboardNavigation/editOnKeyPress.screenshots.ts (136, 7): Cannot find name 'satisfies'.
/Users/william/work/checks/devextreme_Roman/e2e/testcafe-devextreme/tests/dataGrid/keyboardNavigation/editOnKeyPress.screenshots.ts (136, 17): 'Properties' only refers to a type, but is being used as a value here.

```

Reply:
work on my side (c)
`satisfies` is better than `as` because it's type-safe, better not to change it. Can you please check `tsc -v`?